AzureCalling/Controllers/datasource/ParticipantListDataSource.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+struct ParticipantListInfo {
+    let localDisplayName: String
+    let localIsMuted: Bool?
+    let remoteParticipants: [RemoteParticipantInfo]
+}
+
+struct RemoteParticipantInfo {
+    let displayName: String
+    let isMuted: Bool
+}
```

Comment:
Couldn't we just have a list of `ParticipantInfo` that can be used for both local and remote?

Reply:
Good point! That should simplify and generalize the participant list